### PR TITLE
Finish modeling users

### PR DESCRIPTION
**Pull request by hoangnguyen293**
screenshot from 2018-07-09 16-09-49

### CODE REVIEW

**Comment by hoangnguyen293** [comment]
ready

File: app/helpers/users_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (phamvanbk11) [inline app/helpers/users_helper.rb]: Không dùng thì xóa đi nhé

**Comment by phamvanbk11** [comment]
commented

**Comment by hoangnguyen293** [comment]
ready

**Comment by phamvanbk11** [comment]
merged

**Pull request merged**